src/vnc.yml

```diff
@@ -82,6 +96,3 @@
   vars:
     # The username, password, and ssh keys for the VNC user
```

Comment:
Update comment to reflect variables that moved to earlier in this file:
```suggestion
    # The username for the VNC user
```

Reply:
GitHub isn't letting me accept this change right now; it doesn't even show me the buttons to do so.  Hopefully it is just some intermittent GitHub weirdness.

Reply:
Back in the game!